src/App.js

```diff
@@ -1,7 +1,19 @@
 import "./App.css";
 
 function App() {
-  return <div className="App">Hello World</div>;
+  return (
+    <div className="App">
+      <header className="App-header">My cakes</header>
+      <main className="App-main">Content</main>
+      <footer className="App-footer">
+        <ul>
+          <li>MyCakes</li>
+          <li>Plus</li>
+          <li>Favs</li>
+        </ul>
```

Comment:
maybe you would give it a class too? if you want to ever style it or so.

Reply:
good point. I was planning to style it, but I will apply navlink directly at the later stage and style it for sure :)